runtime/util/cphelp.c

```diff
@@ -153,14 +153,14 @@ getModuleJRTURL(J9VMThread *currentThread, J9ClassLoader *classLoader, J9Module
 			}
 		} else {
 			/* its java.base module */
-#define JRT_JAVA_BASE_URL "jrt:/java.base"
-			jrtURL = j9mem_allocate_memory(sizeof(jrtURL->length) + sizeof(JRT_JAVA_BASE_URL) - 1, OMRMEM_CATEGORY_VM);
+			J9_DECLARE_CONSTANT_UTF8(JRT_JAVA_BASE_URL, "jrt:/java.base");
+			const UDATA length = J9UTF8_LENGTH(&JRT_JAVA_BASE_URL);
+			jrtURL = j9mem_allocate_memory(sizeof(((J9UTF8*)0)->length) + length, OMRMEM_CATEGORY_VM);
 			if (NULL == jrtURL) {
 				goto _exit;
 			}
-			strncpy((char *)J9UTF8_DATA(jrtURL), JRT_JAVA_BASE_URL, sizeof(JRT_JAVA_BASE_URL) - 1);
-			J9UTF8_SET_LENGTH(jrtURL, (U_16)sizeof(JRT_JAVA_BASE_URL) - 1);
-#undef JRT_JAVA_BASE_URL
+			memcpy(J9UTF8_DATA(jrtURL), J9UTF8_DATA(&JRT_JAVA_BASE_URL), length); 
+			J9UTF8_SET_LENGTH(jrtURL, length);
```

Comment:
Is this going to cause a warning on some plats, setting a U_16 from a UDATA?

Reply:
Recompiling now to validate the change to `U_16`